h5py/_hl/files.py

```diff
@@ -134,6 +134,19 @@ def make_fapl(driver, libver, rdcc_nslots, rdcc_nbytes, rdcc_w0, **kwds):
     return plist
 
 
+def make_fcpl(track_order=False):
+    """ Set up a file creation property list """
+    if track_order:
+        plist = h5p.create(h5p.FILE_CREATE)
+        plist.set_link_creation_order(
+            h5p.CRT_ORDER_TRACKED | h5p.CRT_ORDER_INDEXED)
+        plist.set_attr_creation_order(
+            h5p.CRT_ORDER_TRACKED | h5p.CRT_ORDER_INDEXED)
+    else:
+        plist = None
```

Comment:
Is there a good reason to return None rather than creating an empty fcpl? The name implies that it's always going to make a property list.

Reply:
I'm not *completely* sure if empty `plist` is the same as `None` for the underlying HDF5 procedures, and I wanted to be maximally-compatible.